config/locales/en.yml

```diff
@@ -91,7 +91,7 @@ en:
         support_url: Support URL
         allow_read_prefs:  read their user preferences
         allow_write_prefs: modify their user preferences
-        allow_write_diary: create diary entries, comments and make friends
+        allow_write_diary: create diary entries, comments and follow users
```

Comment:
This permission is not used now but if it was, would it make sense to bundle follow/unfollow together with diary editing/commenting? Maybe just remove following from here?

Reply:
Agreed as that is never used and there's no good reason to include that Id just drop it from the comment and if we ever want a permission for managing follows we can add that separately.

Reply:
`follow users` part was removed